sig-network/charter.md

```diff
@@ -26,6 +26,7 @@ Some notable examples of areas owned by SIG Network:
 - Developer and user documentation around networking.
 - Help for onboarding new SIG Network members.
 - Grooming network related bugs.
+- IPAM for secondary networks.
```

Comment:
Maybe

    - [`wg-ipam`](link to line in owner_alias file) - IPAM for secondary networks

or

    - IPAM for secondary networks via [`wg-ipam`](link to line in owner_alias file)

Reply:
np, we can add link just after the owner_alias is merged 
created the PR here https://github.com/kubevirt/kubevirt/pull/12058 as was requested